tests/e2e/e2e_setup_test.go

```diff
@@ -30,4 +33,63 @@ var _ = Describe("Cluster preparation", labels.Feature.Cluster, func() {
 			client := rosacli.NewClient()
 			profilehandler.WaitForClusterReady(client, clusterDetail.ClusterID, config.Test.GlobalENV.ClusterWaitingTime)
 		})
+
+	It("Create cluster with invalid volume size [id:66372]",
+		labels.Medium,
+		labels.Day1Negative,
+		func() {
+			minSize := 128
+			maxSize := 16384
+			// Setup
+			client := rosacli.NewClient()
+			rosaCommand, err := config.RetrieveClusterCreationCommand(ciConfig.Test.CreateCommandFile)
+			Expect(err).To(BeNil())
+			if !rosaCommand.CheckFlagExist("--worker-disk-size") {
+				rosaCommand.AddFlags("--worker-disk-size", "300GiB")
+			}
+
+			var values map[string]string
+			var cmd string
+
+			By("Try a worker disk size that's too small")
+			values = map[string]string{
+				"--worker-disk-size": fmt.Sprintf("%dGiB", minSize-1),
+			}
+			rosaCommand.ReplaceFlagValue(values)
+			cmd = rosaCommand.GetFullCommand()
+			fmt.Println(cmd)
```

Comment:
please use Logger or remove that line

Reply:
Woops, forgot to remove that line after I finished debugging a problem